src/ripple/app/data/SociDB.cpp

```diff
@@ -140,84 +148,99 @@ void convert(soci::blob& from, std::vector<std::uint8_t>& to)
     from.read (0, reinterpret_cast<char*>(&to[0]), from.get_len ());
 }
 
-void convert(soci::blob& from, std::string& to)
+void convert (soci::blob& from, std::string& to)
 {
     std::vector<std::uint8_t> tmp;
-    convert(from, tmp);
-    to.assign(tmp.begin (), tmp.end());
+    convert (from, tmp);
+    to.assign (tmp.begin (), tmp.end());
 
 }
 
-void convert(std::vector<std::uint8_t> const& from, soci::blob& to)
+void convert (std::vector<std::uint8_t> const& from, soci::blob& to)
 {
     if (!from.empty ())
         to.write (0, reinterpret_cast<char const*>(&from[0]), from.size ());
 }
 
-int SqliteWALHook (void* s,
-                   sqlite_api::sqlite3*,
-                   const char* dbName,
-                   int walSize)
+namespace {
+
```

Comment:
Its not so much that they are not allowed as that they have less effectiveness in a unity build. If WALCheckpointer is declared in a .cpp file, that's good enough. No need to add additional adornments.